NEWS.md

```diff
@@ -1,4 +1,34 @@
-# mrgsolve (development version)
+# mrgsolve 1.3.0
+
+- The `evdata` object for modeled events now contains a `check_unique` member; 
+  when set to `false`, the event will be processed without checking for a
+  matching record in the modeled event log (#1119). 
+
+- The `amt` attribute in `evdata` modeled event objects is now considered
+  when looking for duplicate records in the modeled event log (#1119).
+
+## Bugs Fixed
+
+- Fixed a bug where multiple lagged doses given at the same time but with 
+  different bioavailability were all given the bioavailability of the 
+  last dosing record (#1129, #1130).
+
+- Fixed a bug where modeled infusions given `now` were never turned
+  off (#1131). 
+
+- Fixed a bug where the `self` object (type: `databox`) could not be 
+  passed into functions written into header files that were included
+  through `$INCLUDE`; these header files are now included immediately
+  preceding any user code written into `$GLOBAL`  (#1125, #1126). 
+
+- Fixed a bug where modeled event log was not getting reset after 
+  simulating the first individual; the resulted in events not getting
```

Comment:
`s/the resulted/this resulted/`?

Otherwise, all of these entries look good to me, and I don't spot any post-1.2.0 user-facing changes that are missing an entry.

Reply:
Thanks.